site/blog/data-poisoning.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_label: Defending Against Data Poisoning Attacks on LLMs—A Comprehensive Guide
+image: /img/blog/data-poisoning/poisoning-panda.jpeg
+date: 2025-01-07
+---
+
+# Defending Against Data Poisoning Attacks on LLMs: A Comprehensive Guide
+
+Data poisoning remains a top concern on the [OWASP Top 10 for 2025](https://owasp.org/www-project-top-10-for-large-language-model-applications/). However, the scope of data poisoning has expanded since the 2023 version. Data poisoning is no longer strictly a risk during the training of Large Language Models (LLMs); it now encompasses all three stages of the LLM lifecycle: pre-training, fine-tuning, and retrieval from external sources. OWASP also highlights the risk of model poisoning from shared repositories or open-source platforms, where models may contain backdoors or embedded malware.
+
+When exploited, data poisoning can degrade model performance, produce biased or toxic content, exploit downstream systems, or tamper with the model’s generation capabilities.
+
+Understanding how these attacks work and implementing preventative measures is crucial for developers, security engineers, and technical leaders responsible for maintaining the security and reliability of these systems. This comprehensive guide delves into the nature of data poisoning attacks and offers strategies to safeguard against these threats.
+
+<!--truncate-->
+
+<figure>
+  <div style={{ textAlign: 'center' }}>
+    <img
+      src="/img/blog/data-poisoning/poisoning-panda.jpeg"
+      alt="Promptfoo Panda as a chemist"
+      style={{ width: '70%' }}
+    />
+  </div>
+</figure>
+
+## Understanding Data Poisoning Attacks in LLM Applications
+
+Data poisoning attacks are malicious attempts to corrupt the training data of an LLM, thereby influencing the model's behavior in undesirable ways. These attacks typically manifest in three primary forms:
+
+1. **Poisoning the Training Dataset**: Attackers insert malicious data into the training set during pre-training or fine-tuning, causing the model to learn incorrect associations or behaviors. This can lead to the model making erroneous predictions or becoming susceptible to specific triggers. They may also create backdoors, where they poison the training dataset to cause the model to behave normally under typical conditions but produce attacker-chosen outputs when presented with certain triggers.
```

Comment:
i know this is a different list from the one in your intro but it is similar enough that it's confusing to me (there's overlap). maybe not an actionable nit but wanted to share.